tests/backends/tests.py

```diff
@@ -388,8 +387,19 @@ def test_no_interpolation_on_sqlite(self):
         # This shouldn't raise an exception
         query = "SELECT strftime('%Y', 'now');"
         connection.cursor().execute(query)
-        self.assertEqual(connection.queries[-1]['sql'],
-            str_prefix("QUERY = %(_)s\"SELECT strftime('%%Y', 'now');\" - PARAMS = ()"))
+        self.assertEqual(connection.queries[-1]['sql'], query)
+
+    @unittest.skipUnless(connection.vendor == 'sqlite',
```

Comment:
`backends` contains a bunch of loosely related tests. They should at a minimum be split in distinct modules and perhaps to their own test apps or to other existing test apps.